sdk/core/Microsoft.Azure.Core.NewtonsoftJson/src/NewtonsoftJsonObjectSerializer.cs

```diff
@@ -27,10 +29,21 @@ public class NewtonsoftJsonObjectSerializer : ObjectSerializer, IMemberNameConve
         private readonly JsonSerializer _serializer;
 
         /// <summary>
-        /// Initializes new instance of <see cref="NewtonsoftJsonObjectSerializer"/>.
+        /// Initializes new instance of <see cref="NewtonsoftJsonObjectSerializer"/>. Uses setting returned by <see cref="CreateJsonSerializerSettings"/>.
+        /// </summary>
+        public NewtonsoftJsonObjectSerializer() : this(CreateJsonSerializerSettings())
+        {
+        }
+
+        /// <summary>
+        /// Returns a <see cref="JsonSerializerSettings"/> that's used when initializing the <see cref="NewtonsoftJsonObjectSerializer"/> using parameterless constructor.
```

Comment:
```suggestion
        /// Returns a <see cref="JsonSerializerSettings"/> that's used when initializing the <see cref="NewtonsoftJsonObjectSerializer"/> using the parameterless constructor.
```
Or maybe just "the default constructor"? Either is fine.